sdk/src/akkaserverless.ts

```diff
@@ -0,0 +1,454 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import * as grpc from '@grpc/grpc-js';
+import * as settings from '../settings';
+
+import * as discovery from '../proto/akkaserverless/protocol/discovery_pb';
+import * as discovery_grpc from '../proto/akkaserverless/protocol/discovery_grpc_pb';
+import * as google_protobuf_empty_pb from 'google-protobuf/google/protobuf/empty_pb';
+import { PackageInfo } from './package-info';
+
+export class Bindings {
+  readonly address: string;
+  readonly port: number;
```

Comment:
Not important, but not sure `Bindings` reads that well. Maybe call the overall class `Address` with `host` and `port` fields/parameters? (also aligned with the env vars)

Reply:
agree 👍